containers/src/Kokkos_OffsetView.hpp

```diff
@@ -220,9 +220,8 @@ class OffsetView : public View<DataType, Properties...> {
   //----------------------------------------
   /** \brief  Compatible view of array of scalar types */
   using array_type =
-      OffsetView<typename traits::scalar_array_type,
-                 typename traits::array_layout, typename traits::device_type,
-                 typename traits::memory_traits>;
+      OffsetView<typename traits::data_type, typename traits::array_layout,
+                 typename traits::device_type, typename traits::memory_traits>;
```

Comment:
This is wrong. The typedef itself needs to be deprecated, not changed.

Reply:
same here.